src/asr/asr_chainer.py

```diff
@@ -146,12 +144,11 @@ def _sum_sqnorm(self, arr):
 
 
 class ChainerMultiProcessParallelUpdaterKaldi(training.updaters.MultiprocessParallelUpdater):
```

Comment:
Probably same. Now you don't have to pass "reader" and you can use built-in `chainer.training.updaters.MultiprocessParallelUpdater` instead of making our own Updater. But I want ask @kan-bayashi for this help.